### DISCUSSION TITLE
Opening a file picker in Wasm

### DISCUSSION THREAD

**Discussion by SafeAt-Malc** [unoplatform/uno#13873] [General]
Hello,

I'm looking to open a file picker to upload image files. I've looked into Xamarin.Forms and tried using mauiessentials as well, but haven't gotten those to play nice with uno. Has anyone had luck getting the file picker to work properly with a Wasm app?

Thank you!

**Comment by limeniye** [comment]
Hello, @SafeAt-Malc! 
I suggest you to check this uno doc
https://platform.uno/docs/articles/features/windows-storage-pickers.html
